src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  width: $calendar-width;
+  padding: 10px;
+
+  &--start-day {
+    @each $day, $day-number in $days-of-week {
+      &--#{$day} :first-child {
+        margin-left: get-offset($day-number, $day-size, $calendar-gap);
+      }
+    }
+  }
+
+  &--month-length {
+    @for $i from 28 through 31 {
+      &--#{$i} :nth-child(n + #{$i + 1}) {
```

Comment:
I think to name variables more clearly is better for understanding what's going on here